drivers/cleanup.py

```diff
@@ -189,8 +191,9 @@ def runAbortable(func, ret, ns, abortTest, pollInterval, timeOut):
                 else:
                     resultFlag.set("failure")
             except Exception, e:
-                resultFlag.set("failure")
                 Util.log("Child process failed with : (%s)" % e)
+                resultFlag.set("failure")
+                Util.logException("This exception has occured")
```

Comment:
This seems like useful debug, If an exception occurs in runAbortable we don't see a stack trace in the log.